quest/plugins/plugins.py

```diff
@@ -103,7 +110,7 @@ def load_providers(update_cache=False):
     settings = get_settings()
 
     if update_cache or plugin_instances['provider'] is None:
-        providers = load_plugins('provider')
+        providers = load_plugins('provider', update_cache=True)
```

Comment:
This goes against the whole point of caching the providers. Why do you want to update the providers list every time they load?